debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/xstream/XstreamStreamingChangeEventSource.java

```diff
@@ -139,4 +159,23 @@ else if (scn != null) {
             throw new RuntimeException(e);
         }
     }
+
+    private void processLCR(LCR lcr, LcrEventHandler handler) throws StreamsException {
+        LOGGER.trace("Receiving LCR");
+
+        // First process the LCR
+        handler.processLCR(lcr);
+
+        // For RowLCR types, if it has chunk data, we should invoke the chunk handlers.
+        // This is to mimic the behavior of the Xstream callback API
+        if (lcr instanceof RowLCR) {
+            if (((RowLCR) lcr).hasChunkData()) {
+                ChunkColumnValue chunk;
+                do {
+                    chunk = xsOut.receiveChunk(XStreamOut.DEFAULT_MODE);
+                    handler.processChunk(chunk);
+                } while (!chunk.isEndOfRow());
+            }
+        }
```

Comment:
We could debate whether this is even necessary given the fact that the callback handler isn't written to support chunks but I opted to include this mainly to remain consistent with what would have be observed by using the callback handler previously if we had encountered a row that had chunk data.